storybookjs/storybook
# Addon-docs: Support for iframeWidth parameter on stories

Issue: https://github.com/storybookjs/storybook/issues/8816

## What I did
Enabled Story components to receive parameter `iframeWidth` to configure the width of the iFrame shown in the Docs page

## How to test
Add parameter iframeWidth to `/storybook/examples/angularcli/src/stories/app.component.stories.ts` like shown below:

![image](https://user-images.githubusercontent.com/48736073/70835611-d3a2b600-1dba-11ea-9eb0-4d83adda5081.png)

Run Angular-cli example, go to `App Component -> Component with separate template -> Docs page` and watch width of the iFrame changed as shown below.

![image](https://user-images.githubusercontent.com/48736073/70835881-b4f0ef00-1dbb-11ea-9494-03f7bd7b332e.png)



- Is this testable with Jest or Chromatic screenshots? ... no
- Does this need a new example in the kitchen sink apps? ... no
- Does this need an update to the documentation? 

If your answer is yes to any of these, please make sure to include it in your PR.
